charts/federator/templates/configmap.yaml

```diff
@@ -40,12 +40,17 @@ data:
 
     {{- with .optSettings }}
     optSettings:
-      setFederationStrategy:
-        {{- if .setFederationStrategy.allowAll }}
+      # Filepath to one or more PEM-encoded server certificates to use as a trust
+      # store when making grpc requests to remote backends
+      {{- if $.Values.remoteCAContents }}
+      remoteCAStore: "/etc/wire/federator/ca/remote-ca.pem"
+      {{- end }}
+      federationStrategy:
```

Comment:
Note: Remember to update the values in the federation anta/bella/chala environments when this PR gets merged.

Reply:
This part should be fine in anta/bella/chala, as by default we use the system CA, and those certificates are supposed to be Let's Encrypt. However, it seems the Let's Encrypt certificates are not actually being used, so I guess there is a problem somewhere in the ingress configuration.